client/components/sites-dropdown/index.jsx

```diff
@@ -74,14 +79,15 @@ export default class SitesDropdown extends PureComponent {
 	render() {
 		return (
 			<div className={ classNames( 'sites-dropdown', { 'is-open': this.state.open } ) }>
+				<QuerySites allSites={ true } />
```

Comment:
This will duplicate sites data in localstorage (SitesLIst) and indexedDB. Let's avoid this for now until we remove more usages of SitesList